zetaclient/chains/bitcoin/signer/signer.go

```diff
@@ -448,40 +450,33 @@ func (signer *Signer) TryProcessOutbound(
 		cancelTx,
 	)
 	if err != nil {
-		logger.Warn().
-			Err(err).
-			Msgf("SignConnectorOnReceive error: nonce %d chain %d", outboundTssNonce, params.ReceiverChainId)
+		logger.Warn().Err(err).Msg("SignWithdrawTx failed")
 		return
 	}
-	logger.Info().
-		Msgf("Key-sign success: %d => %s, nonce %d", cctx.InboundParams.SenderChainId, chain.Name, outboundTssNonce)
+	logger.Info().Msg("Key-sign success")
 
 	// FIXME: add prometheus metrics
 	_, err = zetacoreClient.GetObserverList(ctx)
 	if err != nil {
 		logger.Warn().
 			Err(err).
-			Msgf("unable to get observer list: chain %d observation %s", outboundTssNonce, observertypes.ObservationType_OutboundTx.String())
+			Msgf("unable to get observer list, observation %s", observertypes.ObservationType_OutboundTx.String())
 	}
 	if tx != nil {
 		outboundHash := tx.TxHash().String()
-		logger.Info().
-			Msgf("on chain %s nonce %d, outboundHash %s signer %s", chain.Name, outboundTssNonce, outboundHash, signerAddress)
+		logger.Info().Msgf("signed outboundHash %s signer %s", outboundHash, signerAddress)
```

Comment:
and here

Reply:
https://github.com/zeta-chain/node/pull/3184/commits/5644b3cb1c511f2d5ad34616946fc60bca6d86ef